Gopkg.toml

```diff
@@ -18,6 +18,9 @@ required = [
   source = "https://github.com/fsnotify/fsnotify/archive/v1.4.7.tar.gz"
   name = "gopkg.in/fsnotify.v1"
 
+[[override]]
+  name = "k8s.io/kubectl/pkg/drain"
+  version = "kubernetes-1.13.1"
```

Comment:
There is no version of the kubectl project called `kubernetes-1.13.1` and even if there were, thats not the version we have stored in the vendor directory.

Reply:
fixed.